components/NavigationDrawer/src/MDCBottomDrawerPresentationController.h

```diff
@@ -31,6 +31,21 @@
 - (void)bottomDrawerWillChangeState:
             (nonnull MDCBottomDrawerPresentationController *)presentationController
                         drawerState:(MDCBottomDrawerState)drawerState;
+
+/**
+ This method is called when the drawer is scrolled/dragged and provides a transition ratio value
+ between 0-100% (0-1) that indicates the percentage in which the drawer is close to reaching the end
+ of its scrolling. If the drawer is about to reach fullscreen, its percentage moves between 0-100%
+ as it starts covering the safe area and status bar. If the drawer doesn't reach full screen, it
+ moves between 0-100% as it reaches 20 points away from being fully expanded.
+
+ @param presentationController the bottom drawer presentation controller.
+ @param transitionRatio The transition ratio betwen 0-100% (0-1).
+ */
+- (void)bottomDrawerTopTransitionRatio:
```

Comment:
_Tip:_ `- (void)bottomDrawerTopTransitionRatio:(nonnull MDCBottomDrawerPresentationController *)presentationController transitionRatio:(CGFloat)transitionRatio;` to `- (void)bottomDrawerPresentationController:(nonnull MDCBottomDrawerPresentationController *)presenationController transitionRatio:(CGFloat)transitionRatio;`